index.bs

```diff
@@ -1149,9 +1197,63 @@ The [=new=] {{MLGraphBuilder}} constructor steps are:
+<div class="note">
+    **Arguments:**
+        - *descriptor*: an {{MLOperandDescriptor}} object
+        - *bufferView*: an {{MLBufferView}}
+    **Returns:**: an {{MLOperand}} object.
+</div>
+<div algorithm=constant>
+    The {{MLGraphBuilder/constant(descriptor, bufferView)}} steps are:
+    <div class="note">
+        The permissions and context validity have been checked by [[#api-mlgraphbuilder-constructor]] steps.
+    </div>
+    1. Let |descriptor| be the first argument.
+    1. If |descriptor| is not an an object that [=implements=] {{MLOperandDescriptor}}, then throw a "{{TypeError}}" {{DOMException}} and stop.
+        1. If the [=byte length=] of |descriptor| is not supported by the underlying platform, then throw a "{{DataError}}" {{DOMException}} and stop.
+        1. If the [=check dimensions=] steps given |descriptor|.{{MLOperandDescriptor/type}} and |descriptor|.{{MLOperandDescriptor/dimensions}} return `false`, throw a "{{DataError}}" {{DOMException}} and stop.
+    1. Let |bufferView| be the second argument.
+        1. If invoking <a>validate buffer with descriptor</a> given |bufferView| and |descriptor| return `false`, then throw a "{{TypeError}}" {{DOMException}} and stop.
+    1. Let |operand| be the result of invoking the <a>create MLOperand</a> steps with [=this=], `"constant"` and |descriptor|.
+        1. If that throws, re-throw the exception and stop.
+    1. Let |bytes| be the result of invoking [[=get a copy of the bytes held by the buffer source=]] given |bufferView|.
+    1. Make a request to the underlying platform to register |operand| as a constant with |bytes| as value and store a reference to the corresponding [=implementation-defined=] object to |operand|.{{MLOperand/[[operand]]}}.
+        1. If that fails, throw an "{{OperationError}}" {{DOMException}} and stop.
+    1. Return |operand|.
+</div>
+
+#### The {{MLGraphBuilder/constant(value, type)}} method  #### {#api-mlgraphbuilder-constant-value-type}
+<div class="note">
+    **Arguments:**
+        - *value*: a number
+        - *type*: an optional {{MLOperandType}}, by default *"float32"*.
+    **Returns:**: an {{MLOperand}} object.
+</div>
+<div algorithm=constant-value-type>
+    The {{MLGraphBuilder/constant(value, type)}} steps are:
+    <div class="note">
+        The permissions and context validity have been checked by [[#api-mlgraphbuilder-constructor]] steps.
+    </div>
+    1. Let |value| be the first argument.
+        1. If |value| is not a number, then  throw a "{{TypeError}}" {{DOMException}} and stop.
+    1. Let |type| be the second argument.
+        1. If |type| is `undefined`, let |type| be `"float32"`.
+        1. Otherwise, if |type| is not one of {{MLOperandType}}, then  throw a "{{TypeError}}" {{DOMException}} and stop.
+    1. Let |descriptor| be a new {{MLOperandDescriptor}}.
+        1. Set |descriptor|.{{MLOperandDescriptor/type}} to |type|.
+        1. Set |descriptor|.{{MLOperandDescriptor/dimensions}} to `[ 1 ]`.
```

Comment:
Should we leave `dimensions` undefined? Because the [spec of `MLOperandDescriptor`](https://www.w3.org/TR/webnn/#api-mloperanddescriptor) mentions 

> The dimensions field is only required for tensor operands.

This method actually creates a scalar operand.

Reply:
This step should be removed.

Reply:
Yes, now I got how the alternative plays out. Thanks!